installer_scripts/install-yb-voyager

```diff
@@ -10,12 +10,18 @@ ARGS_LINUX=$@
 
 VERSION="latest"
 
+# Fetch the latest release name, tag name and commit hash from the github api
+LATEST_RELEASE_NAME=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/releases/latest | grep '"name":' | head -n 1 | awk -F '"' '{print $4}')
+LATEST_TAG_NAME=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/releases/latest | grep '"tag_name":' | head -n 1 | awk -F '"' '{print $4}')
+LATEST_TAGGED_COMMIT=$(curl -s https://api.github.com/repos/yugabyte/yb-voyager/git/refs/tags/${LATEST_TAG_NAME} | grep '"sha":' | head -n 1 | awk -F '"' '{print $4}')
+VOYAGER_VERSION=$(echo $LATEST_RELEASE_NAME | sed 's/v//')
```

Comment:
can we log the value of these variables in our log file(`/tmp/voyager.log`)

Reply:
also i see some code repeated here for a certain operation, see if a separate function can be created for the same.
```
grep '"name":' | head -n 1 | awk -F '"' '{print $4}')
```

```
extract_json_value() {
  local key=$1
  grep "\"$key\":" | head -n 1 | awk -F '"' '{print $4}'
}
```

Reply:
I havent added a new function since the changed statements hardly contain any code. But I have added code to push these variables into the LOG file